packages/react-components/src/Concepts/Positioning/PositioningSetTarget.stories.tsx

```diff
@@ -0,0 +1,81 @@
+import * as React from 'react';
+import { Tooltip } from '@fluentui/react-components';
+import { PopperRefHandle } from '@fluentui/react-positioning';
+
+export const ImperativeAnchorTarget = () => {
+  const popperRef = React.useRef<PopperRefHandle>(null);
+  const [open, setOpen] = React.useState(false);
+
+  const onMouseMove = React.useCallback((e: React.MouseEvent) => {
+    const getRect = (x = 0, y = 0) => {
+      return () => ({
+        width: 0,
+        height: 0,
+        top: y,
+        right: x,
+        bottom: y,
+        left: x,
+        x,
+        y,
+      });
+    };
+    const virtualElement = {
```

Comment:
There should be a type for this. Can we re-export it and use it?

Reply:
Yeah I can re-export from react - positioning, we still have the open issue for reexpoting these from the suite

Reply:
actually it's already exported.... I just forgot to use it 🤦